package com.sgg.Exception;

import java.sql.SQLOutput;
import java.util.Date;
import java.util.Scanner;

public class Error_Test {
    public static void main(String[] args) throws Exception {
        //运行时异常
        //越界
//        int[] arr=new int[10];
//        System.out.println(arr[10]);

        //空指针异常
//        String a=null;
//        System.out.println(a.charAt(1));

        //类型错误
//        Object test=new Date();
//        String a=(String) test;

        //数据类型错误
//        String a="abc";
//        int num=Integer.parseInt(a);

        //输入错误
//        Scanner scanner=new Scanner(System.in);
//        int i = scanner.nextInt();
//        System.out.println(i);

        //算术上的错误
//        int a=1;
//        int b=0;
//        System.out.println(a/b);






        
    }
}
